### ISSUE TITLE
numexpr crash after mkl update 

### ISSUE THREAD

**Issue by szs8** [conda/conda#2059]
```python
Python 2.7.11 |Anaconda 2.4.1 (64-bit)| (default, Dec  6 2015, 18:08:32)
Type "copyright", "credits" or "license" for more information.

IPython 4.0.1 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import numexpr
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-f31799291f6e> in <module>()
----> 1 import numexpr

/anaconda2/lib/python2.7/site-packages/numexpr/__init__.py in <module>()
     38 import os, os.path
     39 import platform
---> 40 from numexpr.expressions import E
     41 from numexpr.necompiler import NumExpr, disassemble, evaluate
     42 from numexpr.tests import test, print_versions

anaconda2/lib/python2.7/site-packages/numexpr/expressions.py in <module>()
     43 scalar_constant_types = tuple(scalar_constant_types)
     44
---> 45 from numexpr import interpreter
     46
     47

ImportError: /anaconda2/lib/python2.7/site-packages/numexpr/../../../libmkl_gnu_thread.so: undefined symbol: mkl_pds_ch_blkldlslvs1_pardiso
```

How can I install a non-mkl version of numexpr?

List of related packages -
```
mkl                       11.3.1                        0
mkl-rt                    11.1                         p0
mkl-service               1.1.0                   py27_p0
numexpr                   2.4.4              np110py27_p0  [mkl]
numpy                     1.10.2                  py27_p0  [mkl]
scikit-learn              0.17               np110py27_p1  [mkl]
scipy                     0.16.1             np110py27_p0  [mkl]
```

**Comment by ilanschnell** [comment]
Please see: https://www.continuum.io/blog/developer-blog/anaconda-25-release-now-mkl-optimizations

The error is coming because you have both `mkl-rt` and `mkl` installed.  Please try the following:
```
conda remove mkl-rt
conda install -f mkl
conda install numpy scipy scikit-learn numexpr mkl-service
```

**Comment by szs8** [comment]
Thanks. What is the difference between mkl and mkl-rt?

**Comment by ilanschnell** [comment]
`mkl-rt` is not used anymore, it means MKL-runtime.  And `mkl` used to be the "MKL Optimizations" meta-package.
Now, `mkl` is the runtime, and that's all (no more "MKL Optimizations" meta-package).

**Comment by klankschap** [comment]
after every 
`conda update --all `
it will suggest to install mkl-rt again (and downgrade the other packages)

**Comment by sdvillal** [comment]
In my case, these instructions were not enough to get all packages working, because the accelerate package was forcing numexpr to be kept at oldie 2.4.4, instead of upgrading to 2.5 (and also pandas was linked to some wrong libs). 

While this is fixed, the following worked for me:

```
conda remove mkl-rt accelerate
conda install -f mkl
# note, this reinstalls accelerate but does not downgrade numexpr or reinstall mkl-rt.
# however, do not run conda update --all or conda update anaconda
conda install numpy scipy scikit-learn numexpr pandas numbapro mkl-service
conda update numbapro  
```

Even if I'm an academic, I also think that making mkl free is really sweet. Thanks!

**Comment by klankschap** [comment]
It is nice that mkl is free, but it should not break the garage service procedures.
Not being able to issue
conda update —all
or
conda update anaconda
is a bad thing, no matter how free it comes…

.F

**Comment by scienceopen** [comment]
This issue is also impacting me, upon importing ```tables```.  I get error ending with
```
from numexpr import interpreterImportError: 
$HOME/anaconda3/lib/python3.5/site-packages/numexpr/../../../libmkl_gnu_thread.so: 
undefined symbol: mkl_pds_ch_blkldlslvs1_pardiso
```

As per @ilanschnell I simply typed
``` conda remove mkl-rt```
and the problem was cured, I did not take any other steps.

**Comment by kalefranz** [comment]
The `conda update --all` behavior has been fixed with the `4.0
